java/compaction-job-execution/src/main/java/sleeper/compaction/jobexecution/CompactSortedFilesRunner.java

```diff
@@ -133,7 +142,7 @@ public void run() throws InterruptedException, IOException, ActionException {
                 CompactionJob compactionJob = compactionJobSerDe.deserialiseFromString(message.getBody());
                 LOGGER.info("CompactionJob is: {}", compactionJob);
                 try {
-                    compact(compactionJob, message);
+                    taskFinishedBuilder.addJobSummary(compact(compactionJob, message));
                 } catch (IOException | IteratorException e) {
                     LOGGER.error("Exception running compaction compactionJob", e);
```

Comment:
This log message seems to have had a find & replace mishap at some point. "Exception running compaction job"?